pkg/scripts/poststart/check_fuse_app.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2022 The Fluid Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package poststart
+
+import (
+	"fmt"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	utilpointer "k8s.io/utils/pointer"
+)
+
+const (
+	appScriptName = "app-check-mount.sh"
+	appScriptPath = "/" + appScriptName
+	appVolName    = "check-mount-app"
```

Comment:
How about changing script name  and volume name to `check-dataset-ready.sh` and `check-dataset-ready`